website/source/docs/builders/amazon-ebs.html.md

```diff
@@ -128,6 +128,10 @@ builder.
   Tags and attributes are copied along with the AMI. AMI copying takes time
   depending on the size of the AMI, but will generally take many minutes.
 
+- `region_kms_key_ids` (map of strings) - a map of regions to copy the ami to, 
+  along with the custom kms key id to use for encryption for that region. 
+  Keys must match the regions provided in `ami_regions`
```

Comment:
Do you think you should add a note about how to set a default key and how that won't work if you have snapshot users set?